docs-chef-io/content/inspec/resources/k8s_namespace.md

```diff
@@ -11,57 +11,56 @@ identifier = "inspec/resources/k8s/K8s Namespace"
 parent = "inspec/resources/k8s"
 +++
 
-
 Use the `k8s_namespace` Chef InSpec audit resource to test the configuration of a specific namespace.
 
 ## Installation
 
 ## Syntax
 
 ```ruby
-describe k8s_namespace(name: 'default') do
+describe k8s_namespace(name: 'DEFAULT') do
```

Comment:
Minor thing: **default** is not equal to **DEFAULT**
The same test with name as 'DEFAULT' would fail whereas 'default' would pass